tutorials/clinical/60_sleep.py

```diff
@@ -188,19 +187,18 @@
 fig, (ax1, ax2) = plt.subplots(ncols=2)
 
 # iterate over the subjects
-stages = sorted(event_id.keys())
+stages = sorted(event_id)
```

Comment:
```suggestion
stages = sorted(event_id.keys())
```
Explicit is better than implicit.